.github/workflows/ci-gitlab.yml

```diff
@@ -7,11 +7,17 @@ on:
     branches: [dev, main, github-ci-setup]
     tags:
       - '[0-9]+.[0-9]+.[0-9]+'
+  workflow_run:
```

Comment:
AFAIU, now this workflow will be triggered by 3 events: 1) pull request, 2) tag assigned, 3) other workflow named CI completed. Simple jobs workflow CI is triggered only by pull request created, so, if tag is assigned this workflow is not triggered. I would assume, that assigning tag full CI pipeline has to be executed - simple jobs as well.

Reply:
Thanks, changed